extensions/widgets/graph/docs/guide/guide.md

```diff
@@ -0,0 +1,91 @@
+</svg>
+
+
+Alternatively it can be created in script using:
+
+	create widget as "com.livecode.widget.linegraph"
+
+## Properties
+Unarguably the most important property you will need to know about as 
+you start using the Line Graph is the `graphData`. The `graphData` is 
+comma and line delimited with the first item of each line being a point 
+on the X axis and each subsequent item on a line being a point on the Y 
+axis of a data series.
+
+For example:
+
+	local tData
+	put "Jan,6,22" & return & \
+		"Feb,6,22" & return & \
+		"Mar,8,20" & return & \
+		"Apr,11,17" & return & \
+		"May,14,15" & return & \
+		"Jun,17,12" & return & \
+		"Jul,11,18" & return & \
+		"Aug,18,13" & return & \
+		"Sep,16,15" & return & \
+		"Oct,12,16" & return & \
+		"Nov,9,18" & return & \
+		"Dec,7,20" into tData
+		
+	set the graphData of widget "Line Graph" to tData
+
+![Line graph with data](images/data.png)
+
+The `graphXLines` and `graphYLines` properties control whether the graph
+displays the horizontal and vertical lines respectively. The 
+`markerStyles` property controls the shapes of the data markers. This
+can be set to `empty` to remove the markers entirely. For example, to 
+remove all extraneous parts of the graph other than the data line, use
+the following:
+
+	set the graphXLines of widget "Line Graph" to false
+	set the graphYLines of widget "Line Graph" to false
+	set the markerStyles of widget "Line Graph" to empty
+	
+![Line graph with no grid or markers](images/data-no-markers.png)
+	
+A marker style may be `empty`, `"circle"`, `"diamond"`, `"square"`, 
+`"filled circle"`, `"filled diamond"` or `"filled square"`. 
+The `showLines` property governs the line visibility so that you can 
+just show markers:
+
+	local tMarkers
+	put "filled circle" & return & "filled diamond" into tMarkers
+	set the markerStyles of widget "Line Graph" to tMarkers
+	set the showLines of widget "Line Graph" to false
+
+![Line graph with no grid or markers](images/data-no-lines.png)
+	
+This is helpful for creating a scatter plot.
```

Comment:
Due to the treatment of x axis as labels we can't use the widget as a scatter plot at the moment so perhaps delete this line?